config/serverless.security.yml

```diff
@@ -33,10 +39,6 @@ xpack.fleet.internal.registry.spec.max: '3.0'
 # xpack.fleet.internal.registry.kibanaVersionCheckEnabled: false
 # xpack.fleet.internal.registry.spec.min: '3.0'
 
-# Serverless security specific options
-xpack.securitySolution.enableExperimental:
-  - esqlRulesDisabled
-
```

Comment:
https://github.com/elastic/kibana/pull/167856#discussion_r1347252352

the question was, what's the general purpose of `settings` object - should new values be added here on the root level of `xpack. securitySolution.*`? Since settings name can be seemed very general, so basically anything can be put there.

But after your proposal to change name to offeringsSettings - this question is basically answered

Reply:
I'll take a look to that